Sources/Fluent/FluentProvider.swift

```diff
@@ -128,11 +131,17 @@ extension Application {
         func initialize() {
             self.application.storage[Key.self] = .init(
                 threadPool: self.application.threadPool,
-                on: self.application.eventLoopGroup
+                on: self.application.eventLoopGroup,
+                migrationLogLevel: .info
             )
             self.application.lifecycle.use(Lifecycle())
             self.application.commands.use(MigrateCommand(), as: "migrate")
         }
+        
+        public var migrationLogLevel: Logger.Level {
```

Comment:
@gwynne I am not sure if we should synchronize access to this property. But users will probably only change this during configuration anyhow

Reply:
There are several other properties that'd be affected if we tried to deal with it for this one - it's considered API contract that you can't modify the config once the app is booted (though I'm not sure that's explicitly stated anywhere).